### ISSUE TITLE
Field value list in nested Field Map displays parent fields

### ISSUE THREAD

**Issue by hotsupport** [proofgeist/generator#25]
Just downloaded 2.0.6, testing on FMPA 16.0.3 Mac

Mapping fields for top level response works as expected
Added a nested field map for a response array, but the field value list displays fields from the parent table, not the child table...

Steps to reproduce

1. create a new request whose response will contain a nested array
2. create a new field map
3. select a field and table to map the parent fields
4. click Nested Array in the list of response fields, to create a field map
5. select file and table to map the parent fields
6. go back and work with the parent fields
7. choose the child field map
8. try to map fields

Expected result:
Field value list displays field names from child table

Actual result:
Field value list displays field names from parent table

...I got it to work by reselecting the table for the nested field map:
1. Choose table
2. Refresh file list and select
3. Refresh the table list and select

In conclusion, it is possible to operate Generator 2.0.6 in an way which does not offer the correct value list contents for a nested field map. I suspect it's a missing refresh when navigating back and forth between the two maps
